### DISCUSSION TITLE
Can't install VirtualBox on SLBE 12. Anyone else having this problem?

### DISCUSSION THREAD

**Discussion by birds-swim** [SpiralLinux/SpiralLinux-project#465] [Q&A]
I'm away from my computer right now, but last night I did a fresh install of SLBE 12 (my favorite edition; the glorious edition) and everything went pretty smoothly. No problems with the installer.

So I reboot and get to work building the pillars of my new system just before my first snapshot.

I reboot again and get down to installing VirtualBox. Nothing fancy: just straight from the Stable repos. Well, `dpkg` throws a fit when it gets down to installing the `virtualbox-dkms` package barfing errors everywhere and then my system complains GRUB isn't configured correctly or some stupid thing.

I never touched GRUB. Has anyone else had this much trouble installing VirtualBox? I really need VB because it's waaaaay easier to use than the other options.

It might be this evening until I can give some actual error messages that are helpful to read. Please be patient until then.

**Comment by marwenius** [comment]
Maybe it's the problem that was discussed there and the solution was to install a new version of VirtualBox directly from the manufacturer instead of from the Fasttrack repositories:

https://github.com/orgs/SpiralLinux/discussions/448#discussioncomment-10131781

If you want, you could try my script, which automatically installs the latest version of VirtualBox:

https://github.com/orgs/SpiralLinux/discussions/457

**Reply by birds-swim** [reply]
Thank you for your kind reply. 

I thought I was going to be left out, but I've heard there's a Flatpak called Gnome Boxes.

I will try both VB and Boxes and see which one I like better.

**Reply by marwenius** [reply]
Depending on what your use case is, you could also take a look at [Distrobox](https://packages.debian.org/search?keywords=distrobox):

https://itsfoss.com/distrobox/
https://distrobox.it/
